### DISCUSSION TITLE
HPA is failing

### DISCUSSION THREAD

**Discussion by raosharadhi** [argoproj/argo-helm#2160] [Q&A]
Hi team,
I am using thig argocd version 2.2 and 2.5 in two different clusters. Everything was fine till yesterday, now suddenly when i sync the application. HPA shows the error: "server could not find requested resource kind: autoscaling/v2beta1" and failing the sync. Is this the bug from the argo side? If anyone knows how to solve this please let me know.
Its affecting our higher environment. 
Thanks.

**Comment by mkilchhofer** [comment]
I suspect that your cluster admins upgraded your K8s cluster to 1.25. The autoscaling API `v2beta1` is no longer present in K8s 1.25+:

Ref: https://kubernetes.io/docs/reference/using-api/deprecation-guide/#horizontalpodautoscaler-v125
